client-side(frontend)/lifebird/src/Components1/login.js

```diff
@@ -0,0 +1,43 @@
+import react from 'react';
+import './login.css';
+function Login(){
+    return(
+        <div className="outer-line">
+           <div class="box">
+			<h1 id="head">Login</h1>
+			<div class="logos">
+				<div class="facebook">
+					<img src="facebook.avif"></img>
```

Comment:
use self closing img tag <img src="url"  alt="alternative name"/> in entire document and alt attribute to the image tag